starfish/.dockerignore

```diff
@@ -0,0 +1,4 @@
+*
+!target/*-runner
+!target/*-runner.jar
```

Comment:
Just adding target/ should be sufficient.

Reply:
done